dash/development/_r_components_generation.py

```diff
@@ -323,6 +320,9 @@ def write_help_file(name, props, description, prefix):
 
     has_wildcards = any("-*" in key for key in prop_keys)
```

Comment:
not new here, but `"-*" in key` is different from `key.endswith("-*")` that we use elsewhere, in principle could lead to bugs.

Reply:
fixed in fcd82a52a52584e50f6ceef4d43f9d39df10805f